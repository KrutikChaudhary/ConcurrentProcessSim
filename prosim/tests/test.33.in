2 3
Proc1 3 2
BLOCK 1
DOOP 4
HALT

Proc2 2 1
DOOP 5
HALT
