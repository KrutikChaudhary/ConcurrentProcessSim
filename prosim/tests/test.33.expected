00000: process 1 new
00000: process 1 blocked
00000: process 2 new
00000: process 2 ready
00000: process 2 running
00001: process 1 ready
00003: process 2 ready
00003: process 2 running
00005: process 2 finished
00005: process 1 running
00008: process 1 ready
00008: process 1 running
00009: process 1 finished
Process 1: Run time 4, Block time 1, Wait time 4
Process 2: Run time 5, Block time 0, Wait time 0
